shell/codecov.sh

```
@@ -0,0 +1,30 @@
+# test file of its own, so it always shows 0% and isn't a real gap.
+
+YELLOW='\033[0;33m'
+CYAN='\033[0;36m'
+NC='\033[0m'
+
+GO_PATHS=$(go list -f '{{ .Dir }}' ./... | grep -E -v 'docs|cmd|mocks|internal/fixture')
+
+COVERAGE_FILE=$(mktemp)
+trap 'rm -f "$COVERAGE_FILE"' EXIT
+
+go test -coverprofile="$COVERAGE_FILE" $GO_PATHS > /dev/null 2>&1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)